packages/flutter_webrtc/pubspec.yaml

```diff
@@ -0,0 +1,29 @@
+name: flutter_webrtc_tizen
+description: Flutter WebRTC plugin for Tizen, based on GoogleWebRTC.
+repository: https://github.com/flutter-tizen/plugins/tree/master/packages/flutter-webrtc
+version: 0.9.11
```

Comment:
```suggestion
homepage: https://github.com/flutter-tizen/plugins
repository: https://github.com/flutter-tizen/plugins/tree/master/packages/flutter_webrtc
version: 0.1.0
```

Reply:
Solved

Reply:
A typo in the repository URL:

`flutter-webrtc` → `flutter_webrtc`